### PR TITLE
feat: writed script

### PR DESCRIPTION
**Pull request by tranmarta**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The task was to change the `.box` `border-color` over time.

### Requirements
1. The logic should be written in the `script.js` file.
2. The `setInterval` method should be used with 3 seconds as a parameter.
3. It should change between at least 4 different colors.

### Code Changes Review
The changes in the `script.js` file are as follows:

```diff
@@ -1 +1,3 @@
-console.log('Hello from Nerdbord!')
\ No newline at end of file
+console.log('Hello from Nerdbord!')
+
+setInterval(console.log, 3000);
\ No newline at end of file
```

The changes made in the `script.js` file do not meet the task requirements:

1. The logic is written in the `script.js` file, which meets the first requirement.
2. The `setInterval` method is used, but it is not changing the `.box` `border-color`. It is just logging to the console every 3 seconds. This does not meet the second requirement.
3. There is no code to change between at least 4 different colors, which does not meet the third requirement.

In conclusion, the code changes do not cover the task requirements. The `setInterval` method should be used to change the `.box` `border-color` between at least 4 different colors every 3 seconds.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
Based on the provided task requirements and code changes, I cannot provide a proper code review. The task requirements specify that the code should be written in the `script.js` file and use the `setInterval` method to change the border color of the `.box` class every 3 seconds between at least 4 different colors.

However, the provided code changes only show that a single line of code, `console.log('Hello from Nerdbord!')`, was removed from the `script.js` file. There are no new code changes to review that would fulfill the task requirements. 

Please provide the new code changes that were made to fulfill the task requirements for a proper code review.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

Based on the task requirements, the code changes do not fully meet the requirements. Here are the issues:

1. The `setInterval` method is used with 3 seconds as a parameter, which is correct according to the requirements. However, it is not used correctly. The first argument should be a function to be executed after the specified time interval. In this case, `console.log` is used, which does not make sense in this context.

2. The `changeColor` function is supposed to change the border color of the `.box` element. However, `document.getElementById(".box")` is used, which is incorrect because `.box` is a class, not an id. It should be `document.querySelector(".box")`.

3. The `style.color` property is used to change the color, which changes the text color, not the border color. It should be `style.borderColor`.

4. The `style.color` property is assigned multiple values at once, which is not valid. It should be assigned one value at a time.

5. The color change should occur every 3 seconds, but in the current implementation, it only happens once when the `changeColor` function is called.

6. The code does not ensure that the color changes between at least 4 different colors.

Here's a possible correct implementation:

```javascript
let colors = ["blue", "green", "yellow", "orange"];
let i = 0;

function changeColor() {
    document.querySelector(".box").style.borderColor = colors[i];
    i = (i + 1) % colors.length;
}

setInterval(changeColor, 3000);
```

This code changes the border color of the `.box` element every 3 seconds, cycling through the colors in the `colors` array.

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

**Task goal:** Change the `.box` border-color over time.

**Requirements:**

1. Logic should be written in `script.js` file.
2. `setInterval` method should be used with 3 seconds as a parameter.
3. It should change between at least 4 different colors.

**Review:**

The code changes meet the task requirements:

1. The logic is written in the `script.js` file.
2. The `setInterval` method is used with 3000 milliseconds (which is equivalent to 3 seconds) as a parameter.
3. The border color of the `.box` changes between four different colors: blue, green, orange, and yellow.

The function `changeColor()` changes the border color of the `.box` every time it is called. The `setInterval` function calls this function every 3 seconds.

The variable `i` is used to cycle through the array of colors. After reaching the end of the array, it resets to the beginning due to the modulus operation.

Overall, the code changes cover the task requirements.
